examples/screen_util_example/lib/main.dart

```diff
@@ -43,7 +43,7 @@ Widget exampleBuilder(BuildContext context) {
 ///
 /// For more context on how to create the app builder see [materialAppBuilder]
 /// or have a look at the documentation.
-@anno.WidgetbookAppBuilder()
+@widgetbook.AppBuilder()
```

Comment:
This rename was not necessary because we'll remove the generator soon. This also applies for other annotations, except `WidgetbookUseCase` ➡️ `UseCase`